### ISSUE TITLE
Spelling fixes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8845]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8845|
|Type|enhancement|
|Created|2016-09-29 20:09:51Z|

https://github.com/twisted/twisted/pull/553

<details><summary>Searchable metadata</summary>

```
trac-id__8845 8845
type__enhancement enhancement
reporter__scop scop
priority__low low
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1475179791558432 1475179791558432
changetime__1475901211709912 1475901211709912
version__None None
owner__Craig_Rodrigues__rodrigc_____ Craig Rodrigues <rodrigc@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_2"></a>|@markrwilliams commented|
|-|-|

Thanks!  I'll merge when the builds are green.

**Comment by twisted-trac** [comment]
|<img alt="Ville Skyttä's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|Ville Skyttä commented|
|-|-|

In changeset 781de6cca44b6470f2e1776dc07cfcf5ec78eacd
```
#!CommitTicketReference repository="" revision="781de6cca44b6470f2e1776dc07cfcf5ec78eacd"
Add topfile for #8845
```

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc set owner to @rodrigc<br>@rodrigc set status to `closed`|
|-|-|

In changeset 49365f5c51d0e2dd6a808bfc7dee96946bba81a0
```
#!CommitTicketReference repository="" revision="49365f5c51d0e2dd6a808bfc7dee96946bba81a0"
Merge pull request #556 from twisted/8845-scop-spelling

Author: scop
Reviewer: rodrigc
Fixes: #8845

Spelling fixes
```
